src/Verilator.cpp

```diff
@@ -118,6 +119,7 @@ static void process() {
     V3LinkLValue::linkLValue(v3Global.rootp());
     // Convert return/continue/disable to jumps
     V3LinkJump::linkJump(v3Global.rootp());
+    V3Increments::increments(v3Global.rootp());
```

Comment:
Rename to V3LinkInc?   The early visitors are a bit different so like having Link in the name.